config.go

```diff
@@ -40,9 +41,18 @@ type Config struct {
 	// Tracer holds an optional apm.Tracer to use for tracing bulk requests
 	// to Elasticsearch. Each bulk request is traced as a transaction.
 	//
-	// If Tracer is nil, requests will not be traced.
+	// If Tracer is nil, requests will not be traced. Note however that
+	// OtelTracerProvider may not be nil, in which case the request will
+	// be traced by a different tracer.
 	Tracer *apm.Tracer
 
+	// OtelTracerProvider holds an optional otel TracerProvider for tracing
+	// flush requests.
+	//
+	// If OtelTracerProvider is nil, requests will not be traced.
+	// To use this provider Tracer must be nil.
+	OtelTracerProvider trace.TracerProvider
```

Comment:
Maybe call this `TracerProvider`, with the expectation that we'll remove `Tracer` in the future.

Reply:
done in [`6cc6cea` (#182)](https://github.com/elastic/go-docappender/pull/182/commits/6cc6ceac993cb58d2e7e1fba7ad1ec070a3ca63f)